packages/dashboard/app/components/WorkflowSimpleCanvas.css

```
@@ -0,0 +1,381 @@
+  color: var(--text-muted);
+  width: 28px;
+  height: 28px;
+}
+
+.wf-simple-controls .react-flow__controls-button:hover {
+  background: var(--surface-hover);
+  color: var(--text);
+}
+
+.wf-simple-controls .react-flow__controls-button svg {
+  fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)